modules/mongodb/templates/mongodb-backup-s3.erb

```diff
@@ -17,65 +17,109 @@ date_format = '%Y-%m-%d:%H:%M:%S'
 mongo_server = '<%= @server %>'
 key_fingerprint = '<%= @private_gpg_key_fingerprint %>'
 s3_bucket = '<%= @s3_bucket %>'
-standalone = '<%= @standalone %>'
+<% if standalone -%>
```

Comment:
Sorry, I don't agree. I want the script to do exactly what I tell it to do based on my environment. Environment is handled by hiera. My approach is also consistent with the rest of the module:

```
DaveOps:templates jonauman$ git grep -A5 -E "if @" automongodbbackup
automongodbbackup:<% if @domonthly == false -%>
automongodbbackup-DOMONTHLY=00
automongodbbackup-<% else -%>
automongodbbackup-DOMONTHLY=01
automongodbbackup-<% end -%>
```